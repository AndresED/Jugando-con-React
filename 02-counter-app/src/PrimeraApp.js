import React from 'react';
import './index.css';
const PrimeraApp = ()=>{
    const saludo = <h1> Hola mundo 23</h1>;
    return saludo;
}
export default PrimeraApp;