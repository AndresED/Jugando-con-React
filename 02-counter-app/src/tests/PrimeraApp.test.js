import React from 'react';
import PrimeraApp from '../PrimeraApp';
import '@testing-library/jest-dom';
import { shallow, mount, render } from 'enzyme';
describe('Pruebas sobre PrimeraApp',()=>{
    test("debe de mostrar el mensaje 'Hola, soy Goku'",()=>{
        const saludo = 'Hola, soy Goku';
        const nombre = 'Goku';
        const wrapper = shallow(<PrimeraApp nombre={nombre}/>);
        expect(wrapper).toMatchSnapshot();
    });
});