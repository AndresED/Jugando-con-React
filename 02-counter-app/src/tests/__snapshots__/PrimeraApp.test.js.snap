// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre PrimeraApp debe de mostrar el mensaje 'Hola, soy Goku' 1`] = `
<Fragment>
  <h1>
     Hola, soy 
    Goku
  </h1>
  <h2>
     Â¿Como estas?
  </h2>
  <p>
    Soy un subtitulo
  </p>
</Fragment>
`;
